### ISSUE TITLE
Create Environment object to detail environment and climate specific data points

### ISSUE THREAD

**Issue by odscjen** [open-contracting/infrastructure#393]
There is an increasing requirement to record data points relating to the environmental and climate impact of infrastructure projects as detailed in the summer 2023 OC4IDS review. Suggest the creation of a new `environment` object.

field | type | description | codelist | IDS data point | IDS disclosure format | main issue
 -- | -- | -- | -- | -- | -- | --
hasImpactAssessment | boolean | tbd | NA | Environmental impact category | If environmental impact assessment was not conducted select false | https://github.com/open-contracting/infrastructure/issues/398
impactCategories | array of `Classification` objects | tbd | add ifc-environmental-social to classificationScheme | Environmental impact category | Indicate the category that reflects the magnitude of the environmental impact using the [IFC's Environmental and Social Categorization](https://www.ifc.org/en/what-we-do/sector-expertise/sustainability/policies-and-standards/environmental-and-social-categorization) | https://github.com/open-contracting/infrastructure/issues/398
protectedArea | boolean | tbd | NA | Protected area | Identify whether the project is located in a protected area. Use the WDPA - World Database of Protected Areas | https://github.com/open-contracting/infrastructure/issues/397
conservationMeasures | array of ConservationMeasure objects | tbd | NA | Conservation measures | Disclose and provide further details on the measures adopted by the project to protect and enhance biodiversity. | #401 
ConservationMeasure.type | string | tbd | conservationMeasuresType.csv | | |
ConservationMeasure.description | string | tbd | NA | | |
climateMeasures | array of ClimateMeasure objects | tbd | NA | Climate measures | Clarify whether the project design considered climate change mitigation and/or adaptation measures disclosing the design demonstrating how the measures were incorporated. | #402 
ClimateMeasure.type | string | tbd | climateMeasuresType.csv | | | 
ClimateMeasure.description | string | tbd | NA | | | 
climateChangeGoals | | | | | | #415 
abatementCost | | | | | | #416 

Will create additional tickets to track the development of the 2 new codelists and the new code for classificationScheme as there may be further codes to add to that particular codelist.

cc @EvelynDinora

**Comment by jpmckinney** [comment]
impactClassifications: Feel free to use another word than "classifications" if it would be clearer or more consistent with other terms of art.

protectedArea: In case the protected area itself is described in the future, maybe rename to `inProtectedArea`.

Rest looks good, pending descriptions and codelists in new issues.

**Comment by odscjen** [comment]
I've added a column to the table to indicate where there's another issue to more fully discuss the individual points. This issue is more to show how the various `environment` fields all go together.

**Comment by odscjen** [comment]
Note just waiting on #401 and #402 , all other associated issues now Agreed

**Comment by odscjen** [comment]
All associated issues now in Agreed so moving this there too

**Comment by duncandewhurst** [comment]
Closing as the environment object has been added to the schema.
